packages/cli/src/loaders/index.spec.ts

```diff
@@ -859,6 +859,107 @@ user.password=Contraseña
       );
     });
   });
+  describe('srt bucket loader', () => {
+    it('should load srt', async () => {
+      setupFileMocks();
+  
+      const input = `
+1
+00:00:00,000 --> 00:00:01,000
+Hello!
+
+2
+00:00:01,000 --> 00:00:02,000
+World!
+      `.trim();
+      const expectedOutput = {
+        '0': {
+          startTime: '00:00:00,000',
+          startSeconds: 0,
+          endTime: '00:00:01,000',
+          id: "1",
+          endSeconds: 1,
+          text: 'Hello!',
+        },
+        '1': {
+          startTime: '00:00:01,000',
+          startSeconds: 1,
+          endTime: '00:00:02,000',
+          id: "2",
+          endSeconds: 2,
+          text: 'World!',
+        },
```

Comment:
can you make it work like this:

```suggestion
 '1#00:00:00,000-00:00:01,000': 'Hello!',
 '2#00:00:01,000-00:00:02,000': 'World!',
```
or similar, to make sure we pass forward only the data that needs to be localized?

e.g. we do that for flutter's metadata or for HTML's tags/attributes.

Reply:
on it

Reply:
done